src/app/forms/buttons-page/buttons-page.component.ts

```diff
@@ -21,12 +21,12 @@ export class ButtonsPageComponent {
     autoUpload: false
   });
 
-  onClick(msg: string, targetId?: string) {
+  onClick(msg: string, targetId?: string, successProbability: number = 0.5) {
```

Comment:
Nitpick: If we we merge the eslint PR first this might get flagged to be changed to : `successProbability = 0.5`